config/xsl/loadreport/sections/errors.xsl

```diff
@@ -216,20 +216,18 @@
 				                            	</xsl:attribute>
 												<xsl:attribute name="alt">Errors by Type</xsl:attribute>
 											</img>
-											<a class="backLink">
-												<xsl:attribute name="href">
+											<a class="backlink"><xsl:attribute name="href">
 													<xsl:value-of select="concat('#tableEntry-',id)" />
-												</xsl:attribute>
-												Back to table
+												</xsl:attribute>Back to Table
 											</a>
 										</div>
 									</div>
 								</xsl:for-each>
 							</div>
 						</xsl:when>
 						<xsl:otherwise>
-							<p class="text-center">
-								There are no charts to show.
+							<p class="text-center no-data">
```

Comment:
Outside tables, `no-data` does currently not provide any specific styling.

Reply:
Fixed. no-data is not a p with styling.